examples/createtool/createtool.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <title>Cindy JS</title>
+        <style>
+            body {
+                margin: 0;
+                padding: 0;
+            }
+
+            button {
+                background-color: #FFFFFF;
+                border: 1px solid #CCCCCC;
+            }
+        </style>
+        <script type="text/javascript" src="../../build/js/Cindy.js"></script>
+        <script type="text/javascript">
+            var cindy = createCindy({ 
+        	scripts: "cs*", 
+        	defaultAppearance: {}, 
+        	geometry: [], 
+        	ports: [ 
+        		{ id: "CSCanvas", width: 680, height: 336, transform: [ { visibleRect: [ -9.04, 9.32, 18.16, -4.12 ] } ], background: "rgb(168,176,192)" } ] });
```

Comment:
@gagern wrote:
>If I understand you correctly, you're currently saying that Cinderella is failing to recreate its own port sizes. Do you know of a Cinderella issue filed for this? I see no urgent need to fix this here, since we don't really care about which portion of the plane we see; it's all empty anyway.

This was reported 6 months ago as [#88 Save/Close/Load cycle shrinks every window by 14 pixels in height](https://gitlab.cinderella.de:8082/cinderella/driftin/issues/88)